backend/routes/auth.js

```diff
@@ -212,10 +204,14 @@ router.post('/password-resetcode', async (req, res) => {
     //generate code for verification
     const codeReset = generateVerificationCode();
     user.verificationCode = codeReset;
+    // verification code expiration time
```

Comment:
nit: add a comment to explain how long the expiration time is (it may not be immediately clear what 60000 means)